### ISSUE TITLE
Root-paths must start with a "/" or be a wild-card After migration to v6 

### ISSUE THREAD

**Issue by Raider-hub** [Milad-Akarie/auto_route_library#1708]
### Root-paths must start with a "/" or be a wild-card.

i used the Auto router helper to migrate, everything looks fine but my app wont run
![Screenshot 2023-08-29 114121](https://github.com/Milad-Akarie/auto_route_library/assets/85285046/bf9c1122-4882-4482-a207-a50ad8c34d8d)
![Screenshot 2023-08-29 114131](https://github.com/Milad-Akarie/auto_route_library/assets/85285046/24bfc9e1-db23-4d12-a8c1-5711ccbb98a8)

![Screenshot 2023-08-29 114330](https://github.com/Milad-Akarie/auto_route_library/assets/85285046/7ebea272-2aeb-421c-8323-ef1ba8c20581)

**Comment by Milad-Akarie** [comment]
@Raider-hub you need to make all root paths start with '/<path', this was the case before v6 but the check happened in generation time.

**Comment by Raider-hub** [comment]
Please how do I go about fixing it now, I don't understand your explanation.

**Comment by vanyasem** [comment]
@Raider-hub prefix all paths by `/`

For example, change
`AutoRoute(path: 'contacts', page: ContactsRoute.page)`
to
`AutoRoute(path: '/contacts', page: ContactsRoute.page)`

Repeat that for all paths

**Comment by vanyasem** [comment]
Duplicate of #1450 and #1534
